examples/step-29/step-29.cc

```diff
@@ -330,31 +329,25 @@ namespace Step29
   {}
 
 
-  // The actual postprocessing happens in the following function.  Its inputs
-  // are a vector representing values of the function (which is here
-  // vector-valued) representing the data vector given to
-  // DataOut::add_data_vector, evaluated at all evaluation points where we
-  // generate output, and some tensor objects representing derivatives (that
-  // we don't use here since $|u|$ is computed from just $v$ and $w$, and for
-  // which we assign no name to the corresponding function argument).  The
-  // derived quantities are returned in the <code>computed_quantities</code>
-  // vector.  Remember that this function may only use data for which the
-  // respective update flag is specified by
+  // The actual postprocessing happens in the following function. Its input is
+  // an object that stores values of the function (which is here vector-valued)
+  // representing the data vector given to DataOut::add_data_vector, evaluated
+  // at all evaluation points where we generate output, and some tensor objects
+  // representing derivatives (that we don't use here since $|u|$ is computed
+  // from just $v$ and $w$). The derived quantities are returned in the
+  // <code>computed_quantities</code> vector. Remember that this function may
+  // only use data for which the respective update flag is specified by
   // <code>get_needed_update_flags</code>. For example, we may not use the
   // derivatives here, since our implementation of
   // <code>get_needed_update_flags</code> requests that only function values
   // are provided.
   template <int dim>
   void
-  ComputeIntensity<dim>::compute_derived_quantities_vector (
-    const std::vector<Vector<double> >                 &solution_values,
-    const std::vector<std::vector<Tensor<1, dim> > >   & /*solution_gradients*/,
-    const std::vector<std::vector<Tensor<2, dim> > >   & /*solution_hessians*/,
-    const std::vector<Point<dim> >                     & /*normals*/,
-    const std::vector<Point<dim> >                     & /*evaluation_points*/,
-    std::vector<Vector<double> >                       &computed_quantities
-  ) const
+  ComputeIntensity<dim>::compute_derived_quantities_vector
+  (const DataPostprocessorInputs::Vector<dim> &inputs,
+   std::vector<Vector<double> >               &computed_quantities) const
   {
+    const std::vector<Vector<double> > &solution_values = inputs.solution_values;
```

Comment:
I find this alias more confusing than helpful. Would you mind just using the argument instead of the alias?

While you're there, please also wrap line 363 :-)